index.html

```diff
@@ -58,8 +57,207 @@ <h3>I am a Software Developer</h3>
+            <h3>Art Printing Data</h3>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul id="sp-lang-wrapper">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <a id="sp-link" href="#">
+          <button id="sp-button">See Project</button>
+        </a>
+      </div>
+
+      <div id="simple-post">
+        <div id="sp-content">
+          <div id="sp-h-wrapper">
+            <h3>Art Printing Data</h3>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul id="sp-lang-wrapper">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <a id="sp-link" href="#">
+          <button id="sp-button">See Project</button>
+        </a>
+      </div>
+
+      <div id="simple-post">
+        <div id="sp-content">
+          <div id="sp-h-wrapper">
+            <h3>Art Printing Data</h3>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul id="sp-lang-wrapper">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <a id="sp-link" href="#">
+          <button id="sp-button">See Project</button>
+        </a>
+      </div>
+
+      <div id="simple-post">
+        <div id="sp-content">
+          <div id="sp-h-wrapper">
+            <h3>Art Printing Data</h3>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul id="sp-lang-wrapper">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <a id="sp-link" href="#">
+          <button id="sp-button">See Project</button>
+        </a>
+      </div>
+
+      <div id="simple-post">
+        <div id="sp-content">
+          <div id="sp-h-wrapper">
+            <h3>Art Printing Data</h3>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul id="sp-lang-wrapper">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <a id="sp-link" href="#">
+          <button id="sp-button">See Project</button>
+        </a>
+      </div>
+
+    </div>
+  </div>
+</section>
+
+<section id="about-me">
+  <div id="am-content">
+    <h2 id="am-heading">About me</h2>
```

Comment:
- [ ] Great job so far. please ensure that the about me title is centralized 
- [ ] please add the `see my resume` button so it follows the figma size
yours
![Screenshot (463)_LI](https://user-images.githubusercontent.com/65298048/192093822-5084d9ad-59fa-4763-8014-eadcd13bc89b.jpg)
figma
![Screenshot (464)](https://user-images.githubusercontent.com/65298048/192093825-ae31d1f2-8313-4f76-bf98-02e571b70fe4.png)

Reply:
In the comment I noted that you need to set the width of a viewport to 375. 
1. Heading. As you see in the screenshot (Figma on the left, my page on the right), the heading in Figma is not centralized and so do I, my looks the same.
3. Button. During project implementation I tried to follow the specs as much as I can, so, if you check my previous commit and Figma, you'll notice that I entered everything as it's in Figma.
But, I don't know why it really seems smaller than Figma's one. So, here I enlarged it.
![2022-09-24_17-25-55](https://user-images.githubusercontent.com/48753073/192096698-e86dd9ec-d4b6-468e-a1b2-24992f862341.png)